File: Front/Proizvod.js
export class Proizvod{
    constructor(id, nazivProizvoda, cena, kolicina){
        this.id = id;
        this.nazivProizvoda = nazivProizvoda ?? "Proizvod 1";
        this.cena = cena ?? 0;
        this.kolicina = kolicina;
        //this.tip = tip;

        this.container = null;
    }

}

File: Models/Proizvod.cs
using System.ComponentModel.DataAnnotations;
namespace Models{
    public class Proizvod{
        [Key]
        public int ID{get; set;}
        public string? NazviProizovda{get; set;}
        public float Cena{get; set;}
        public int Kolicina{get; set;}
        public Tip? Tip{get; set;}
    }
}